diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/LineLength.kt

```diff
@@ -156,18 +157,20 @@ class LineLength : Rule("line-length") {
             }
             val nodeText = "//${commentText.substring(indexLastSpace, commentText.length)}"
             val newNode = LeafPsiElement(EOL_COMMENT, nodeText)
-            node.treeParent.addChild(LeafPsiElement(EOL_COMMENT, commentText.substring(0, indexLastSpace)), node)
-            node.treeParent.addChild(PsiWhiteSpaceImpl("\n"), node)
-            node.treeParent.addChild(newNode, node)
-            node.treeParent.removeChild(node)
+            node.treeParent.run {
+                addChild(LeafPsiElement(EOL_COMMENT, commentText.substring(0, indexLastSpace)), node)
+                addChild(PsiWhiteSpaceImpl("\n"), node)
+                addChild(newNode, node)
+                removeChild(node)
+            }
             node = newNode
         } while (node.text.length > configuration.lineLength)
     }
 
     private fun fixCondition(wrongNode: ASTNode, configuration: LineLengthConfiguration,
                              leftInitOffset: Int = -1, isProperty: Boolean = false) {
         var leftOffset = if (leftInitOffset < 0){
-            positionByOffset(wrongNode.firstChildNode.psi.textOffset).second
+            positionByOffset(wrongNode.firstChildNode.psi.startOffset).second
```

Comment:
I meant just `ASTNode.starOffset`, without referencing `PsiElement`

Reply:
I thought I deleted it